src/scenarioLoading.ts

```diff
@@ -46,11 +46,15 @@ export const loadAllScenarios = (
                     )}-${_i}`;
                     const ctx = { scenario: scenarioNameWithIdx };
 
-                    const actionCatalogWithReplacedLoadVariables: Action[] = [];
+                    const actionCatalogWithReplacedLoadVariables: Action[] = JSON.parse(
+                        JSON.stringify(actionCatalog),
+                    );
+                    /*
```

Comment:
If we really want to remove this, remove it instead of commenting.

Reply:
sure!